livy/README.md

```diff
@@ -0,0 +1,26 @@
+# Apache Livy Initialization Action
+
+This [initialization action](https://cloud.google.com/dataproc/init-actions) installs version
+`0.5.0` of [Apache Livy](https://livy.incubator.apache.org/) on a master node within a
+[Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
+
+## Using this initialization action
+You can use this initialization action to create a new Dataproc cluster with Livy installed by:
+
+1. Uploading a copy of this initialization action (`livy.sh`) to
```

Comment:
Heh I know a lot of init action README's have instructions to copy a script to your own GCS bucket. I'd rather have all of them avoid that and just have a `clusters create` command that uses our mirror of this repo on GCS:

```
gcloud dataproc clusters create <CLUSTER_NAME> \
  --initialization-actions gs://dataproc-initialization-actions/livy/livy.sh
```

Reply:
Done